Makefile

```diff
@@ -32,18 +32,80 @@ LDS_FLAGS	:= -DSVSM_GPA_LDS="$(SVSM_GPA)"
 LDS_FLAGS	+= -DSVSM_GPA="$(SVSM_GPA)ULL"
 LDS_FLAGS	+= -DSVSM_MEM="$(SVSM_MEM)ULL"
 
-.PHONY: all doc prereq clean superclean
-
-all: .prereq svsm.bin
+EXT_LIBS = external/libcrt/libcrt.a
+EXT_LIBS += external/openssl/libcrypto.a
+
+.PHONY: all doc prereq clean superclean external
+
+all: external .prereq svsm.bin
+
+external:
+	$(MAKE) -C external/libcrt
+	(cd external/openssl && git checkout OpenSSL_1_1_1q  && \
```

Comment:
@Zildj1an Good idea. I think it may be useful to have them as separate target `prereq_external`.

```
 prereq: prereq_rust prereq_external
     touch .prereq
 
 prereq_rust:
     curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
     source $(HOME)/.cargo/env
     echo "source $(HOME)/.cargo/env" >> ~/.bashrc
     rustup component add rust-src
     rustup component add llvm-tools-preview
     cargo install xargo
     cargo install bootimage
 
 prereq_external:
     git submodule update --init
      (cd external/openssl && git checkout OpenSSL_1_1_1q  && \
          ./Configure \
              --config=../openssl-svsm.conf \
              SVSM \
              no-afalgeng \
```